src/SpacePlanning/Generate/Packers/CuboidPacker.cs

```diff
@@ -221,6 +223,27 @@ private static List<Item> ItemsFromCuboids(List<Cuboid> cuboids)
             return items;
         }
 
+        private static List<Cuboid> TransformPackedCuboids(Cuboid container, List<Cuboid> packedItems)
```

Comment:
name should be a bit more descriptive : `TransformPackedCuboidsToContainerCoords` maybe ?

Reply:
resolved in #134